tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -72,6 +127,28 @@ def test_vm_clone(
                 f"Failed: MD5 comparison between source {vm_obj.name} "
                 f"and cloned {clone_obj.name} VMs"
             )
+            # Expand PVC if `pvc_expand_after_restore` is True
+            if pvc_expand_after_clone:
+                new_size = 50
+                try:
+                    clone_pvc_obj = clone_obj.get_vm_pvc_obj()
+                    clone_pvc_obj.resize_pvc(new_size=new_size, verify=True)
+                    assert (
+                        clone_pvc_obj.get_vm_pvc_obj().size == new_size
+                    ), f"Failed: VM PVC Expansion on cloned VM {clone_obj.name} "
+                    time.sleep(30)
+                    result = clone_obj.run_ssh_cmd(command="lsblk -o SIZE")
+                    if str(new_size) in result:
+                        log.info("expanded PVC size is showing on vm")
+                    else:
+                        raise ValueError(
+                            "Expanded PVC size is not showing on VM. "
+                            "Please verify the disk rescan and filesystem resize."
+                        )
+                except ValueError as e:
+                    log.error(
+                        f"Error for VM {vm_obj}: {e}. Continuing with the next VM."
+                    )
```

Comment:
shouldn't we fail the test instead of allowing it to run for next iteration ?

Reply:
here I am collecting vms which are failed in list. So for running test for next VM i did not failed it here.